react/components/EXPERIMENTAL_Table/constants.ts

```diff
@@ -34,6 +54,7 @@ export const EMPTY_STATE_SIZE_IN_ROWS = 5
 export const DEFAULT_SCROLLBAR_WIDTH = 17
 export const FIELDS_BOX_ITEM_HEIGHT = 36
 export const NESTED_ROW_PREFIX_WIDTH = 36
+export const SELECTED_ROW_BACKGROUND = '#dbe9fd'
```

Comment:
New color?

Reply:
This color is defined like that on the `Table v1`. I did not found a className for it on `tachyons` or `design-tokens` 🙁

Reply:
Better update to a token that will be close then.

Reply:
Updated to `bg-action-secondary`, it is very alike